app/assets/javascripts/controllers/projects/EditController.js

```diff
@@ -0,0 +1,20 @@
+import TagGroupsController from 'controllers/tag_groups/TagGroupsController';
+
+export default () => {
+  TagGroupsController();
+  
+  const nameConfirmation = $('#name_confirmation')
+  const projectDeleteLink = $('#project-delete')
+
+  nameConfirmation.keyup(() => {
+    verifyProjectName()
+  })
+    
+  function verifyProjectName() {
+    if (nameConfirmation.data('project-name') === nameConfirmation.val()){
+      projectDeleteLink.attr("disabled", false)
```

Comment:
This won't always work, you should this to be sure
```suggestion
      projectDeleteLink.removeAttr("disabled")
```